extensions/amp-analytics/0.1/events.js

```diff
@@ -44,11 +44,16 @@ const TAG = 'amp-analytics/events';
  * @enum {string}
  */
 export const AnalyticsEventType = {
-  VISIBLE: 'visible',
+  AMP_STORY: 'amp-story',
```

Comment:
Given now all `TRACKER_TYPE` is listed in the AnalyticsEventType now. 
Could you please change the `#isReservedTriggerType()` method as well.

Reply:
done

Reply:
I didn't see the change. Meant the `#isReservedTriggerType()` method implementation.

Reply:
now added. also included a test